cyber/tools/cyber_monitor/renderable_message.h

```diff
@@ -57,7 +57,7 @@ class RenderableMessage {
   virtual RenderableMessage* Child(int /* lineNo */) const = 0;
 
   virtual double frame_ratio(void) { return frame_ratio_; }
-  double band_width(void) {return band_width_;}
+  double band_width(void) {return band_width_; }
```

Comment:
Why did I just add a space, CI build reported an error?

Reply:
CI also checks the lint errors.

Reply:
CI error is not a link error，the error message is as follows:
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

Reply:
Seeing that the information is CI build timeout, how can CI be rebuilt again?

Reply:
@jinghaomiao